Event Delegation/app.js

```diff
@@ -0,0 +1,9 @@
+let inputItems = document.querySelectorAll('input');
+
+for (let i = 0; i < inputItems.length; i++) {
+	inputItems[i].addEventListener('click', (e) => {
+		if (e.target.tagName === 'INPUT') {
+			e.target.style.backgroundColor = 'rgb(255, 255, 0)';
+		}
+	});
+}
```

Comment:
Please format codes properly.

Reply:
Thanks, @lycoangelo, This is now updated.

Reply:
Thanks, @lycoangelo, This is now updated.